patroni/validator.py

```diff
@@ -1124,6 +1124,7 @@ def validate_watchdog_mode(value: Any) -> None:
             Optional("ports"): [{"name": str, "port": IntValidator(max=65535, expected_type=int, raise_assert=True)}],
             Optional("cacert"): str,
             Optional("retriable_http_codes"): Or(int, [int]),
+            Optional('location_cache_ttl'): IntValidator(min=0, max=600, base_unit='s', raise_assert=True)
```

Comment:
Not sure if that kicks in at any time during reload, but the name is wrong anyway
```suggestion
            Optional('xlog_cache_ttl'): IntValidator(min=0, max=600, base_unit='s', raise_assert=True)
```